rules/windows/persistence_ms_outlook_vba_template.toml

```diff
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2020/11/23"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/23"
+
+[rule]
+author = ["Elastic"]
+description = "Detects attempts to establish persistence on an endpoint by installing a rogue Microsoft Outlook VBA Template."
```

Comment:
Maybe explain what VBA stands for and what it's used for to put it in context. I think that's optional, so going to approve regardless.

Reply:
details are in the 2 reference blogs